src/monopoly.js

```diff
@@ -4,10 +4,10 @@ import Player from './player';
 import fs from 'fs';
 import events from 'events';
 import randomjs from 'random-js';
-import {standardProperty, railroad, utility, eventCard, noEvent, go, goToJail, incomeTax, luxuryTax} from './landOnFunctions';
+import {StandardProperty, Railroad, Utility, EventCard, NoEvent, Go, GoToJail, IncomeTax, LuxuryTax} from './properties';
```

Comment:
If you don't want to import all these, you could alternatively have './properties' export that PROPERTY_TYPES object.   Up to you.  It's just a matter of convenience.